tools/index.js

```diff
@@ -0,0 +1,35 @@
+const admin = require('firebase-admin');
+
+var serviceAccount = {
```

Comment:
Why you use firebase-admin and not firebase ? I think [firebase-admin](https://firebase.google.com/docs/admin/setup) provides more privileges but maybe useless in our use case. 

I recommend to not save on github credentials, but use instead fake values, correct values could be retrieved on firebase console if required.

Reply:
That was recommended by firestore [docs](https://firebase.google.com/docs/firestore/quickstart?hl=pt#set_up_your_development_environment ) for node.js